test/Microsoft.AspNetCore.Server.Kestrel.Performance/RequestParsing.cs

```diff
@@ -23,6 +23,11 @@ public class RequestParsing
 
         private const string plaintextRequest = "GET /plaintext HTTP/1.1\r\nHost: www.example.com\r\n\r\n";
 
+        private const string plaintextTechEmpower = "GET /plaintext HTTP/1.1\r\n" +
```

Comment:
Let's keep it for now till we make the same change to dev-si and reset the baselines.